src/main/java/com/epam/grid/engine/mapper/job/slurm/SlurmJobMapper.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *
+ *  * Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ *
+ */
+
+package com.epam.grid.engine.mapper.job.slurm;
+
+import com.epam.grid.engine.entity.job.Job;
+import com.epam.grid.engine.entity.job.JobState;
+import com.epam.grid.engine.entity.job.slurm.SlurmJob;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+
+import java.util.Set;
+
+/**
+ * This interface consists of the configuration of a mapper for implementation by the MapStruct processor
+ * for creation a {@link Job} object out of a {@link SlurmJob} object.
+ * To get more information about grid engine job status values see <a href="https://manpages.debian.org/testing/gridengine-common/sge_status.5.en.html">
+ * Grid Engine job status values</a>.
+ */
+@Mapper(componentModel = "spring")
+@ConditionalOnProperty(name = "grid.engine.type", havingValue = "SLURM")
+public interface SlurmJobMapper {
+
+    String finishedStatusCode = "COMPLETED";
```

Comment:
1. Those fields are `final` and `static` so should be considered constants - use capital letters for its naming.
2. All the fields, except the `errorStatusCodeList` fits line limit (120 chars) - keep them in one line without breaking the last element of collection onto a new one.

Reply:
fixed